Libraries/CO.CDP.Authentication.Tests/Authorization/OrganisationScopeAuthorizationHandlerTests.cs

```diff
@@ -71,8 +69,8 @@ public async Task HandleRequirementAsync_MatchExpectedResult(bool organisationSc
         var context = CreateAuthorizationHandlerContext(userUrn, organisationScopeExists ? ["Admin"] : ["Scope1"], location);
         MockHttpContext(location, string.Format(urlFormat, organisationId));
 
-        _mockTenantRepository.Setup(x => x.LookupTenant(userUrn))
-            .ReturnsAsync(GetTenantLookup(userUrn, organisationId));
+        _mockOrganisationRepository.Setup(x => x.FindOrganisationPerson(organisationId, userUrn))
+            .ReturnsAsync(GetOrganisationPerson());
```

Comment:
I'm not sure if that's right. We're suppose to use the tenant lookup for any authorisation requests.

Reply:
The lookup endpoint is going to be very expensive as it builds complex query and brings lots of information which is not required in authorisation scope check, I have created another version of lookup which creates a nominal query.

Reply:
Again this is not exposed as endpoint but an internal repo method, lets have a call to discuss why you think its not right?